### DISCUSSION TITLE
Search by topic does not work

### DISCUSSION THREAD

**Discussion by Sofitt** [community/community#145849] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

We have some badges with topics

![image](https://github.com/user-attachments/assets/08292cb7-d9b9-48cc-b026-fb09cdb69611)

But after clicking on it, I get a 404 page.

![image](https://github.com/user-attachments/assets/f85671a9-2986-43ab-9281-48b7a94f7b76)

Finally, I can't find any repository with the same topic, because of the error

Topic example:
[https://github.com/topics/unixporn](https://github.com/topics/unixporn)

**Comment by Naz33m** [comment, marked as answer]
Hey @Sofitt   Use the regular search bar and type 
topic:your_topic like topic:python.

### If it still gives 404:
- Check for typos.
- Clear browser cache.

See [githubstatus.com](https://www.google.com/url?sa=E&q=https%3A%2F%2Fwww.githubstatus.com%2F).

**Comment by deoxys3** [comment]
I'm a year late, but it's because the topic has the word "p*rn" in it, you won't be able to search for any topic that include certain words
